easypost/requestor.py

```diff
@@ -222,7 +224,7 @@ def requests_request(
         except Exception as e:
             raise Error(
                 "Unexpected error communicating with EasyPost. If this "
-                f"problem persists please let us know at {SUPPORT_EMAIL}.",
```

Comment:
We want to keep the `f` here because it will populate the `SUPPORT_EMAIL` with the right data: https://peps.python.org/pep-0498/

Reply:
@ryannguyen16 please address this and we should be good to stamp it.